src/main/java/io/georgeous/mcgenerations/commands/admin/ModRenameCommand.java

```
@@ -31,49 +30,75 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
             return true;
         }
 
-        if (!player.hasPermission("1h1l.commands.rename")) {
+        if (!player.hasPermission("1h1l.commands.modrename")) {
             Notification.noPermission(player);
             return true;
         }
 
         if (args.length == 0) {
-
             return true;
```

Why return true if the command was used incorrectly? This will silently fail. Return false to have it print the command usage.